engines/tantivy-0.6/src/bin/build_index.rs

```diff
@@ -25,21 +28,31 @@ fn main_inner(output_dir: &Path) -> tantivy::Result<()> {
     let schema = create_schema();
     let index = Index::create_in_dir(output_dir, schema.clone()).expect("failed to create index");
 
-    // 4 GB heap
-    let mut index_writer = index.writer(200_000_000).expect("failed to create index writer");
+    {
+        // 4 GB heap
+        let mut index_writer = index.writer(1_500_000_000).expect("failed to create index writer");
```

Comment:
the comment is (and was) erraneous. Also do we need 1.5GB?

Reply:
not sure, I just made it consistent with tantivy-0.7

Reply:
You mean it was still memory per thread in 0.6 ? If so, this is still dependant on the number of cpu cores we have right?

Reply:
oh, this changed in 0.7? ok, I will put it back to 200_000_000